tests/run-tests.js

```diff
@@ -18,19 +16,17 @@ const testDirs = fs
   .filter(test => fs.statSync(test.path).isDirectory());
 
 const results = testDirs.map((test) => {
-  const argsPath = path.join(test.path, 'args.json');
+  const inputPath = path.join(test.path, 'input.txt');
 
-  const args = fs.existsSync(argsPath)
-    ? require(argsPath)
-    : [];
+  const rawInput = fs.readFileSync(inputPath, 'utf-8');
+  const input = rawInput.trim().split(' ');
 
-  const mainFile = path.join(test.path, 'Main.elm');
-  const expectedOutput = fs.readFileSync(path.join(test.path, 'output.txt'));
+  const expectedOutput = fs.readFileSync(path.join(test.path, 'output.txt'), 'utf-8');
 
   let actualOutput;
 
   try {
-    actualOutput = spawnSync(runElmPath, [mainFile, ...args]).stdout;
+    actualOutput = spawnSync(runElmPath, input, { cwd: test.path }).stdout.toString();
```

Comment:
`cwd` is being changed to allow for relative paths in command arguments (e.g. `Main.elm` instead of `/path/to/Main.elm`)